### PR TITLE
digitalocean: create as many nodes as specified

### PR DESCRIPTION
**Pull request by s-urbaniak**

### CODE REVIEW

**Comment by s-urbaniak** [comment]
@sttts PTAL

File: provider/cloud/digitalocean/provider.go [hunk 1]
```diff
@@ -110,78 +110,83 @@ func (do *digitalocean) CreateNodes(
...
-	glog.V(2).Infof("dropletName %q", dropletName)
+		glog.V(2).Infof("dropletName %q", dropletName)
 
-	clientKC, err := cluster.CreateKeyCert("dropletName")
-	if err != nil {
-		return nil, err
-	}
+		clientKC, err := cluster.CreateKeyCert("dropletName")
+		if err != nil {
+			return created, err
```
Comment (sttts) [inline provider/cloud/digitalocean/provider.go:127]: indention?

File: provider/cloud/digitalocean/provider.go [hunk 2]
```diff
@@ -110,78 +110,83 @@ func (do *digitalocean) CreateNodes(
...
-		ClientKey:         clientKC.Key.Base64(),
-		ClientCert:        clientKC.Cert.Base64(),
-		RootCACert:        cluster.Status.RootCA.Cert.Base64(),
-		ApiserverPubSSH:   cluster.Status.ApiserverSSH,
-		ApiserverToken:    cluster.Address.Token,
-		FlannelCIDR:       cluster.Spec.Cloud.Network.Flannel.CIDR,
-	}
+		image := godo.DropletCreateImage{Slug: "coreos-stable"}
+		data := ktemplate.Data{
+			DC:                spec.DC,
```
Comment (sttts) [inline provider/cloud/digitalocean/provider.go:132]: indention?

File: provider/cloud/digitalocean/provider.go (hunk 1 above)
Comment (s-urbaniak) [inline provider/cloud/digitalocean/provider.go:127]: there's a for loop now.

File: provider/cloud/digitalocean/provider.go (hunk 2 above)
Comment (s-urbaniak) [inline provider/cloud/digitalocean/provider.go:132]: it's in a for loop now.

**Pull request merged**